includes/class-woocommerce.php

```diff
@@ -29,7 +29,7 @@ final class WooCommerce {
 	 *
 	 * @var string
 	 */
-	public $db_version = '430';
+	public $db_version = '440';
```

Comment:
you don't need to update DB version every release but only if there is a change in any of the base table.